src/templates/readme.md

```diff
@@ -6,6 +6,13 @@ Use this repo to bootstrap Adobe API Mesh development with ease. Use this repo t
 
 This repo comes with all the files and dependencies necessary to get started with API Mesh.
 
+1. `mesh.json` - sample mesh config with a single source
+2. `.env` - environment file with variables for the config presented in mesh.json
+3. `package.json` - dependencies and scripts to test and deploy meshes
+4. `.vscode/launch.json` - VS Code configuration to setup debugging out of the box. This applies to VS Code on local machines or Codespaces on Github.com
+5. `.devcontainer/devcontainer.json` - Codespaces configuration to setup dev container out of the box. This config will help setup the packages and build the mesh config automatically. As a bonus it also sets up port-forwarding so you can use your favourite GraphQL interface to practice the mesh
+6. `.github/workflows/deployMesh.yml` - Github workflow to automatically publish mesh config when something is committed to `main`
```

Comment:
Based on the name of the file, should this be:
```suggestion
6. `.github/workflows/deployWorkflow.yml` - Github workflow to automatically publish mesh config when something is committed to `main`
```

Reply:
Its `deployMesh.yaml` after the `init` process is completed but good that you bought it up. I had to update the `init` command to name the file accordingly. It was naming it `deploy.yaml` instead of `deployMesh.yaml`.